portfolio/src/main/java/com/google/sps/servlets/DeletionServlet.java

```diff
@@ -82,13 +88,22 @@ private String getParameter(HttpServletRequest request, String name, String defa
   private class DeletionRequestBody {
     // The only parameter in the body is the id of the comment to be deleted.
     private long id;
+    private String imageBlobstoreKey;
 
     public long getId() {
       return this.id;
     }
 
+    public String getImageBlobstoreKey() {
+        return this.imageBlobstoreKey;
+    }
+
     public String toString() {
-      return String.format("DeletionRequestBody{ id=\"%d\" }", this.id);
+      return String.format(
+          "DeletionRequestBody{ id=\"%d\", imageBlobstoreKey = %s }",
+          this.id,
+          this.imageBlobstoreKey
```

Comment:
Is this private class provided to serialise an incoming deletion request, or to send one out?
In either case, can Gson not be used instead of manually formatting the JSON string?

Reply:
toString() isn't used when handling requests, it was just used in a manual debugging context. I'll remove it now though as it's unused.

Reply:
removed now 👍 https://github.com/thomasogara/step-repository/pull/48/commits/d6f89f7bc0ccf83a58065fa3ed9ab12925f332d4